File: ts/wam/6746.ts
export const WamSmbQpCallHealth = 6746 as const

export interface SmbQpCallHealthEvent {
  /** field 1, wire `fetch_exception_message` */
  fetchExceptionMessage?: string
  /** field 2, wire `fetch_mechanism` */
  fetchMechanism?: FETCHMECHANISMENUM
  /** field 3, wire `fetch_result` */
  fetchResult?: FETCHRESULTENUM
}

File: go/wam/6746.go
const WamSmbQpCallHealth = 6746 // channel: regular

type SmbQpCallHealthEvent struct {
	FetchExceptionMessage string `wam:"fetch_exception_message"` // field 1
	FetchMechanism FETCHMECHANISMENUM `wam:"fetch_mechanism"` // field 2
	FetchResult FETCHRESULTENUM `wam:"fetch_result"` // field 3
}